CONTRIBUTING.md

```diff
@@ -492,6 +518,12 @@ We rely on renovate to update dependencies automatically.
 
 </details>
 
+<details><summary>Where to find the issues to contribute?</summary>
+
+[This](https://github.com/freeCodeCamp/chapter/contribute) is a good place to go if you are looking to get started with beginner friendly issue.
```

Comment:
Nit pick on the link text since we tend to avoid "click here" and other generic words.  How about "The repository's contribute page" instead of "This"